### PR TITLE
Added `addSegmentationToViewWithID:` and `addSegmentationToViewWithName:` in view interface

### PR DESCRIPTION
**Pull request by ijunaid**

### CODE REVIEW

File: CountlyViewData.h [hunk 1]
```diff
@@ -35,6 +35,10 @@
  */
 @property (nonatomic) BOOL isAutoPaused;
 
+
+
```
Comment (turtledreams) [inline CountlyViewData.h:39]: maybe comments here

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ## 23.8.4
 * Added `disableLocation` initial config property to disable location tracking
+* Added `addSegmentationToViewWithID:` in view interface
```
Comment (turtledreams) [inline CHANGELOG.md:3]: + " for adding segmentation to an ongoing view"

File: CountlyViewTrackingInternal.m [hunk 3]
```diff
@@ -348,6 +348,11 @@ - (void)stopViewWithIDInternal:(NSString *) viewKey customSegmentation:(NSDictio
         segmentation[kCountlyVTKeyName] = viewData.viewName;
         segmentation[kCountlyVTKeySegment] = CountlyDeviceInfo.osName;
         
+        if (viewData.segmentation)
```
Comment (turtledreams) [inline CountlyViewTrackingInternal.m:351]: 1.what will happen if someone calls 'stopViewWithName' or stopallviews or if its an autostoppedview? comes back to here?

**Review by turtledreams — CHANGES_REQUESTED** [review]

File: CountlyViewTrackingInternal.m (hunk 3 above)
Comment (ijunaid) [reply CountlyViewTrackingInternal.m:351]: Yes, In all stop view cases this the central place.

**Review by turtledreams — APPROVED** [review]

**Review by peterBrxwn — APPROVED** [review]

**Pull request merged**